jsonresp/error_test.go

```diff
@@ -29,13 +29,21 @@ func TestError_Respond(t *testing.T) {
 		wantBody    string
 	}{
 		{
-			desc:        "basic",
+			desc:        "basic | string",
 			given:       jsonresp.Error(errors.New("invalid id"), "Invalid ID provided", http.StatusBadRequest),
 			wantCode:    http.StatusBadRequest,
 			wantHeaders: nil,
 			wantCookies: nil,
 			wantBody:    `{"error":"Invalid ID provided"}`,
```

Comment:
this is fine, and backwards compatible: app codebase prefer the convenience and lightweight nature of just saying the error message `string`, library wrap it in `{ "error": ??? }`